classes/cyclonedx-export.bbclass

```diff
@@ -67,23 +68,28 @@ python do_cyclonedx_package_collect() {
             return
 
     # get all CVE product names and version from the recipe
-    name = d.getVar("CVE_PRODUCT")
-    version = d.getVar("CVE_VERSION")
```

Comment:
should it not be enough to assign the version here once (outside the for loop)? AFAIK, CVE_VERSION can only be defined once per recipe (https://docs.yoctoproject.org/singleindex.html#term-CVE_VERSION), thus it will always be assigned the same value within the recipe.